src/main/webapp/WEB-INF/web.xml

```diff
@@ -64,16 +64,6 @@ http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
         <url-pattern>/appstats/*</url-pattern>
     </servlet-mapping>
 
-    <security-constraint>
-        <web-resource-collection>
-            <web-resource-name>appstats</web-resource-name>
-            <url-pattern>/appstats/*</url-pattern>
-        </web-resource-collection>
```

Comment:
Ah, my mistake. This part doesn't belonged to Remote API Servlet. I will revert it.